src/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs

```diff
@@ -48,6 +56,7 @@ public void SslStream_StreamToStream_Authentication_Success()
             return false;
         }
 
+        [ActiveIssue(3362, PlatformID.AnyUnix)]
```

Comment:
We can remove this if https://github.com/dotnet/corefx/pull/3449 gets merged before this PR